publication-doi-commons/src/main/java/no/unit/nva/publication/doi/PublicationMapper.java

```diff
@@ -19,10 +22,13 @@
 import no.unit.nva.publication.doi.dynamodb.dao.DynamodbStreamRecordJsonPointers.DynamodbImageType;
 import nva.commons.utils.JsonUtils;
 
+import static java.util.function.Predicate.not;
+
 public class PublicationMapper {
```

Comment:
Add a short comment about the functionality of this class

Reply:
Added